README.md

```diff
@@ -97,7 +97,7 @@ litgpt finetune lora \
   --checkpoint_dir checkpoints/microsoft/phi-2 \
   --data JSON \
   --data.json_path my_custom_dataset.json \
-  --val_split_fraction 0.1 \
+  --data.val_split_fraction 0.1 \
```

Comment:
thanks!

@rasbt @lantiga @Andrei-Aksionov can we put all the commands from the readme into CI/CD? we don't want these failing....